graph.go

```diff
@@ -306,9 +311,13 @@ func doGraph(buildPkg *build.Package) (*gog.Output, error) {
 	allGoFiles = append(allGoFiles, buildPkg.GoFiles...)
 	allGoFiles = append(allGoFiles, buildPkg.CgoFiles...)
 	allGoFiles = append(allGoFiles, buildPkg.TestGoFiles...)
-	if len(allGoFiles) == 0 {
-		return &gog.Output{}, nil
+
+	if !testPkg {
+		// graph non-test package
+		return doGraphFiles(fset, buildPkg.ImportPath, buildPkg.Dir, allGoFiles, dependencies)
 	}
+
+	// prepare type info for non-test package
```

Comment:
We calculate the type info of the non-test package here, because it is required as a dependency for the analysis of the test package, see line 340. Any suggestion on how to improve the comment to make that clearer?